docs/book/how-to/develop-locally/keep-your-pipeline-dashboard-clean.md

```diff
@@ -0,0 +1,35 @@
+---
+description: Learn how to keep your pipeline runs clean during development.
```

Comment:
maybe worth adding the zenml artifact prune option?

Reply:
done